apis/mlops/scheduler/scheduler.proto

```diff
@@ -401,6 +401,8 @@ message PipelineStatusRequest {
 
 message PipelineSubscriptionRequest {
   string subscriberName = 1; //Name of the subscription caller
+  string subscriberIp = 2; // IP of the subscription caller
```

Comment:
At the time the scheduler receives the subscription request, this info is probably accessible directly via `https://pkg.go.dev/google.golang.org/grpc/peer#FromContext` applied to the grpc context.

Reply:
Were you considering complex scenarios such as multiple pipeline subscribers (potentially pipeline-gateway pods) connecting via a single proxy? (in that case, the `FromContext` would only return the IP of the proxy).